gcloud/pubsub/_gax.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+try:
+    from google.gax import CallOptions
+except ImportError:
+    _HAVE_GAX = False
+else:
+    _HAVE_GAX = True
+
+
+    from google.gax.errors import GaxError
```

Comment:
@tseaver Can you just put this in a separate module where it isn't conditionally defined? (e.g. the pattern [used for crypto][1] implementations in `oauth2client`)

[1]: https://github.com/google/oauth2client/blob/e99a4ac3c5d3c2944680ea180b71a0d144d6128b/oauth2client/crypt.py#L46-L71